"0","popula√ß√£o_municipiosTESTE5 <- faixa_masculino |> 
  group_by(ibge) |> 
  summarise(popula√ß√£o = sum(total))
"
"1","[1m[33mError[39m in `summarise()`:[22m
[38;5;232m[36m‚Ñπ[38;5;232m In argument: `popula√ß√£o = sum(total)`.
[36m‚Ñπ[38;5;232m In group 1: `ibge = ""5200050""`.[39m
[1mCaused by error in `sum()`:[22m
[33m![39m 'type' inv√°lido (character) do argumento
Backtrace:
[90m 1. [39m[1mdplyr[22m::summarise(group_by(faixa_masculino, ibge), popula√ß√£o = sum(total))
[90m 2. [39m[1m[94mdplyr:::summarise.grouped_df([39m[22mgroup_by(faixa_masculino, ibge), popula√ß√£o = sum(total)[1m[94m)[39m[22m
[90m 3. [39mdplyr:::summarise_cols(.data, dplyr_quosures(...), by, ""summarise"")
[90m 5. [39mdplyr:::map(quosures, summarise_eval_one, mask = mask)
[90m 6. [39m[1mbase[22m::lapply(.x, .f, ...)
[90m 7. [39m[1mdplyr[22m (local) FUN(X[[i]], ...)
[90m 8. [39mmask$eval_all_summarise(quo)
[90m 9. [39m[1mdplyr[22m (local) eval()
"
